docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -80,3 +80,14 @@ This can be used in combination with `componentsProps`.
 In the example below, the actions are the same as in the section above, but they are rendered inside a menu:
 
 {{"demo": "ActionBarComponent.js"}}
+
+## Paper content
+
+The paper content is available on all desktop picker components.  
```

Comment:
I don't know what is added at the end of the line, but it creates a `<br/>` in the generated HTML which is a bit weird

![image](https://user-images.githubusercontent.com/45398769/186158061-d9c3407a-16dd-43c6-80ea-85ea580de733.png)


```suggestion
The paper content is available on all desktop picker components.
```

Reply:
I've added [two spaces](https://www.w3schools.io/file/markdown-line-break/#:~:text=The%20fourth%20way%2C%20use%20two%20spaces%20with%20entering%20or%20return%20keystrokes.), which creates a line break, but not a separate paragraph.

Do you disagree with the current visual formatting and would like to reduce the amount of new lines?

Reply:
It sounds a bit like bullet points, but without the bullets

I think you can group the two first one because they answer the same question "what is it" and put the two other in a distinct paragraph since they are more about technical details